### DISCUSSION TITLE
Zigbe Aalliance going Connectivity Standards Alliance and Matter

### DISCUSSION THREAD

**Discussion by MattWestb** [zigpy/zigpy#737] [General]
New name of the Alliance and also for the network system connect IP is renamed after many turns to Matter.

Still interesting then Zigbee is going native over IP that shall being possible with (Open)Thread.

We wile see what is bringing and I still waiting for getting one ZCL v8 for our devs ;-)

**Comment by Hedda** [comment]
You read the news too. I just posted news about Connectivity Standards Alliance (CSA) and Matter in Home Assistant community forum:

https://community.home-assistant.io/t/zigbee-alliance-renamed-to-the-connectivity-standards-alliance-csa-and-project-chip-officially-branded-as-matter-and-given-a-logo-to-prepare-for-product-availability-later-this-year/307350

# Zigbee Alliance renamed to the Connectivity Standards Alliance (CSA) and Project CHIP officially branded as "Matter" and given a logo to prepare for product availability later this year

Zigbee Alliance has announced that the IoT (Internet-of-Things) technology consortium organization has been renamed to "Connectivity Standards Alliance" (or "CSA" for short) to make it clearer that is governing more IoT standards than just the Zigbee standard, (those standards include; Zigbee, Smart Energy, Green Power, JupiterMesh, Dotdot, and now also Matter).

https://zigbeealliance.org/news_and_articles/connectivity-standards-alliance/

The IoT-over-IP working group "Project Connected Home over IP" (a.k.a. "Project CHIP", i.e. a new high-level IP based IoT network connectivity standard that will be supported over Thread and WiFi and possibly more networking media technologies in the future) has now also officially been named "Matter" as its final wordmark/brand name as well as given an official logo/symbol in advance of the first Matter-branded devices which should available before the end of 2021.

https://zigbeealliance.org/news_and_articles/chip-is-now-matter/

The new official website for Matter is **[buildwithmatter.com](https://buildwithmatter.com/)** (as the old website [www.connectedhomeip.com](https://www.connectedhomeip.com) is no longer online) and the project's source code git repositories on GitHub is still active at https://github.com/project-chip/ and Connectivity Standards Alliance new website https://csa-iot.org/ still redirect to https://zigbeealliance.org/about/

Much more information posted about this news on CSA and Matter https://zigbeealliance.org/about/newsroom/

https://youtu.be/5YckrbbqUlY

https://youtu.be/hWYzoUoXFy4

https://youtu.be/bnHbJLMGgTs

https://youtu.be/dOF85994n28

https://zigbeealliance.org/wp-content/uploads/2021/05/Matter-Family.mp4

https://zigbeealliance.org/wp-content/uploads/2021/05/CSA_Image-Lifestyle-1.png

https://www.dropbox.com/sh/oyqcikuvzfhbxo8/AADtWT_MAk7Y6Hl0KEy3_oF4a?dl=0

Members poised to be early adopters of Matter include: Amazon, ASSA ABLOY, Comcast, Espressif Systems, Eve Systems, Google, Grundfos Holding A/S, Huawei, Infineon Technologies, LEEDARSON, Legrand, Nanoleaf, Nordic Semiconductor, NXP Semiconductors, Qorvo, Resideo, Schlage, Schneider Electric, Philips Hue / Signify, Silicon Labs, Samsung SmartThings, Somfy, STMicroelectronics, Texas Instruments, Tuya Smart, ubisys, Wulian, and Zumtobel Group.

For reference, I first read about this news on Engadget website myself:

https://www.engadget.com/chip-rebrand-matter-221410229.html

Engadget in turn refer to this press release as their source of this news:

https://www.businesswire.com/news/home/20210511005928/en/The-Connectivity-Standards-Alliance-Unveils-Matter-Formerly-Known-as-Project-CHIP

![image](https://user-images.githubusercontent.com/6320001/117949965-fa984b00-b312-11eb-97c2-e89741a1cc1b.png)

**Reply by MattWestb** [reply]
I was looking if IKEA have doing more certification of firmware yesterday evening and was finding one new logo and was start reading.

And its start moving in the community to implanting Thread and boarder router for using with Zigbee.
de(F)CONZ is interested and Z2M is defiantly not interested.
Getting Zigbee being routed  between networks sounds great and its possible with all current lards SoC that is being used to day.

The question is is ZHA  interested or only AH ??

**Reply by Hedda** [reply]
I think it would be great to have a stand-alone Zigbee-to-Matter application based on zigpy (i.e. a "zigpy2matter" project).

So a project somewhat similar to Zigbee2MQTT concepts but instead a "Zigbee2Matter" Zigbee to Matter bridge/gateway.

In a similar idea it would also be great to also have a "zwavejs2matter" project that would be similar to zwavejs2mqtt:

https://github.com/zwave-js/zwavejs2mqtt
